src/librustc_mir/transform/instrument_coverage.rs

```diff
@@ -192,10 +195,13 @@ fn placeholder_block(span: Span) -> BasicBlockData<'tcx> {
     }
 }
 
-fn hash_mir_source<'tcx>(tcx: TyCtxt<'tcx>, def_id: DefId) -> u64 {
+fn hir_body<'tcx>(tcx: TyCtxt<'tcx>, def_id: DefId) -> &'_ rustc_hir::Body<'_> {
```

Comment:
I believe all of the elided lifetimes here are also `'tcx` so it's probably clearer to either elide all of the lifetimes or explicitly make them all `'tcx`.

Reply:
Fixed. Thanks!